src/Consumer.php

```diff
@@ -60,9 +60,9 @@ public function __construct($options = null)
      * @return Laminas\OAuth\Token\Request
      */
     public function getRequestToken(
-        array $customServiceParameters = null,
-        $httpMethod = null,
-        Http\RequestToken $request = null
+        ?array $customServiceParameters = null,
+        ?string $httpMethod = null,
```

Comment:
Even though this is the correct type, it's a BC break to add the native paramter type to a class that is open to inheritance

Reply:
@gsteel How did you check this BC Report?

Reply:
@manishranjan-adobe see https://3v4l.org/Fd7CZ , if non-typed changed to param type, child will error

Reply:
non-typed param doesn't require nullable implicit type, so please rollback as is as before, it will keep working on php 8.4 https://3v4l.org/sJa9i/rfc#vgit.master